pandas/core/resample.py

```diff
@@ -205,7 +205,7 @@ def __getattr__(self, attr: str):
 
     # error: Signature of "obj" incompatible with supertype "BaseGroupBy"
     @property
-    def obj(self) -> NDFrameT:  # type: ignore[override]
+    def obj(self) -> NDFrameT:  # type: ignore[override, type-var]
```

Comment:
You can probably replace `NDFrameT` with `NDFrame`?

Reply:
Yes, true. Let me do that.